woodwork/tests/fixtures/datetime_freq.py

```diff
@@ -50,8 +50,8 @@
 def pad_datetime_series(dates, freq, pad_start=0, pad_end=100):
     dates = [pd.Timestamp(d) for d in dates]
 
-    head = pd.Series([])
-    tail = pd.Series([])
+    head = pd.Series([], dtype="float64")
+    tail = pd.Series([], dtype="float64")
```

Comment:
Should this be `datetime64[ns]` here instead of `float64`?

Reply:
Was unsure here. The current default dtype of an empty series in Pandas is `float64`, so this made explicit what was being tested currently. I was unsure if setting it to `datetime64[ns]` would somehow lead to us not testing the scenario where the series dtype was `float64`, if that was important.

Reply:
Probably doesn't matter too much, but it looks like everything ends up getting converted to `datetime64[ns]` somewhere downstream of this anyway, so maybe it makes sense to start with that to be clear we expect these series to contain datetime values.